SPEC.md

```diff
@@ -65,21 +65,21 @@ Examples: `minitest`, `mysql2`, `battery-horse`.
 
 The version number of the library (String, required).
 
-- It should follow the [Semantic Versioning](http://semver.org/) format.
+- It should follow the [Semantic Versioning 2.0](http://semver.org/spec/v2.0.0.html) format.
```

Comment:
Ah, wasn't aware of that development decision. I suspected this might have merited its own issue to discuss it rather than be bolted onto this PR, but I suppose that already happened.

Given the discussion in #10 and that this isn't pertinent to the GitLab sugar, I'll propose the changes in a separate PR.